webapp_v2/src/components/studies/CreateStudyModal.tsx

```diff
@@ -61,6 +66,7 @@ function CreateStudyModal(props: PropTypes) {
         const sid = await createStudy(studyName, vrs, group);
         const metadata = await getStudyMetadata(sid);
         await changePublicMode(sid, publicMode);
+        if (tags.length > 0) await updateStudyMetadata(sid, { tags });
```

Comment:
combine metadata with tags:  
```
{...metadata, tags}
```
Because updateStudyMetadata replace the whole metadata

Reply:
well, you have to sort this, (use only necessary data and skip the name)